pages/photos.js

```diff
@@ -0,0 +1,968 @@
+import Layout from '../components/Layout.js';
+
+export default () => (
+    <Layout> 
+        <div id="fh5co-page">
+            <a href="#" className="js-fh5co-nav-toggle fh5co-nav-toggle"><i></i></a>
+            <aside id="fh5co-aside" role="complementary" className="border js-fullheight">
+
+                <h1 id="fh5co-logo"><a href="index.html">
+                    <img src="../static/logo-colored.png" alt=""/>
+                    </a>
+                </h1>
+                <nav id="fh5co-main-menu" role="navigation">
+                    <ul>
+                        <li className="fh5co-active"><a href="#">Home</a></li>
+                        <li><a href="#">Photos</a></li>
+                        <li><a href="#">About</a></li>
+                        <li><a href="#">Contact</a></li>
+                    </ul>
+                </nav>
+
+                <div className="fh5co-footer">
+                    <p>
+                        <small>&copy; 2018 PeraSports, UOP. All Rights Reserved.
+                            <span>Designed by <a href="http://freehtml5.co/" target="_blank">Engineering Faculty, UOP</a> </span>
+                            <span>Images: <a href="http://pexels.com/" target="_blank">Pera Beats</a></span>

```

Comment:
We can add perabeats fb page

Reply:
@shyaman Added the link, resolving the conversation